src/pykiso/lib/connectors/cc_grpc_client.py

```diff
@@ -0,0 +1,216 @@
+        spec = importlib.util.spec_from_file_location(
+            Path(generated_protobuf_grpc_file).stem, Path(generated_protobuf_grpc_file)
+        )
+        self.grpc_stub_module = importlib.util.module_from_spec(spec)
+        spec.loader.exec_module(self.grpc_stub_module)
+        # Extract the stubs
+        module_attributes = inspect.getmembers(self.grpc_stub_module, inspect.isclass)
+        self.grpc_stub = []
+        for attribute in module_attributes:
+            if attribute[0].endswith("Stub"):
+                self.grpc_stub.append(attribute[1])
+        # Prepare the empty stub list. It will be used for binding the stubs to the channel
+        self.stubs = {}
+
+        # Set the default rpc and message
+        self.default_service_name = default_service_name
+        self.default_rpc_name = default_rpc_name
+        self.default_message_name = default_message_name
+        self.default_param = default_param
+        self.list_of_received_messages = []
+
+    def _cc_open(self) -> None:
+        """Open the grpc connection to server."""
+        self.channel = grpc.insecure_channel(self.dest_ip + ":" + str(self.dest_port))
+        # Bind stubs to new channel
+        self.stubs = dict(
+            [(stub.__name__[:-4], stub(self.channel)) for stub in self.grpc_stub]
+        )
+        # Validate at least that most of the default parameters are valid
+        try:
+            if self.default_service_name not in self.stubs:
+                raise ValueError(f"{self.default_service_name=} was not found!")
+            if not hasattr(
+                self.stubs[self.default_service_name], self.default_rpc_name
+            ):
+                raise ValueError(f"{self.default_rpc_name=} was not found!")
+            if not hasattr(self.grpc_datamodel, self.default_message_name):
+                raise ValueError(f"{self.default_message_name=} was not found!")
+            message_class = getattr(self.grpc_datamodel, self.default_message_name)
+            message_class(**self.default_param)
+        except TypeError as e:
+            self._cc_close()
+            raise ValueError(f"{self.default_param=} was not valid!") from e
+        except Exception as e:
+            self._cc_close()
+            raise e
+
+    def _cc_close(self) -> None:
+        """Close the grpc connection to server."""
+        self.stubs.clear()
+        self.channel.close()
+
+    def _cc_send(self, msg: bytes, **kwargs) -> None:
+        """Send message to grpc server.
+
+        :param msg: message to send, should bytes.
+
+        In addition to the message, the following keyword arguments are supported:
+
+            - rpc_name: name of the rpc to trigger
+            - message_name: name of the message to use
+            - message_attributes: attributes to use in dictionary format
+
+        """
+        # Extract needed parameters or the default once
+        service_name = kwargs.get("service_name", self.default_service_name)
```

Comment:
As these keywords arguments are extremely specific to this connector, I still have troubles to understand how this connector can be used by a generic auxiliary 
Maybe an example would help!

Reply:
https://kiso-testing.readthedocs.io/en/latest/advanced_usage/how_to_connector.html#parameters-and-return-values

I added an example but I did not test it yet, I will do that eventually (before we merge this PR. Nevertheless, I think you can take a look at it and tell me what you think.

I may need to extend the com-aux to support the feature I need. I thought it was already part of it